docker/Dockerfile.build

```diff
@@ -0,0 +1,64 @@
+FROM ubuntu:18.04
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+WORKDIR /usr/src
+
+RUN set -ex \
+	&& apt-get update \
+	&& apt-get -y dist-upgrade \
+	&& apt-get install -qq --no-install-recommends \
+	git ca-certificates \
+	&& rm -rf /var/lib/apt/lists/*
+
+RUN git clone https://github.com/OrchidTechnologies/orchid.git
```

Comment:
A shallow clone might be a bit faster and take up less space. We don't really need the full git history if we are only building the latest commit.

Reply:
At least on my connection it saves space but not much time although happy to save either so changed it to `--depth 1`